core/json/Iden3PaymentRailsRequestV1.json

```diff
@@ -0,0 +1,68 @@
+{
+  "EIP712Domain": [
+    {
+      "name": "name",
+      "type": "string"
+    },
+    {
+      "name": "version",
+      "type": "string"
+    },
+    {
+      "name": "chainId",
+      "type": "uint256"
+    },
+    {
+      "name":  "verifyingContract",
+      "type:": "address"
+    }
+  ],
+  "Proof": [
+    {
+      "name": "created",
+      "type": "string"
+    },
+    {
+      "name": "eip712",
+      "type": "Eip712"
+    },
+    {
+      "name": "proofPurpose",
+      "type": "string"
+    },
+    {
+      "name": "proofValue",
+      "type": "string"
+    },
+    {
+      "name": "type",
+      "type": "string"
+    },
+    {
+      "name": "verificationMethod",
+      "type": "string"
+    }
+  ],
```

Comment:
```suggestion
 
```

Reply:
let's removre `Proof` type from here, 
even if it's mentioned in documentation, `"type": "Eip712"` is not valid.

Also tried with ethers `signTypedData` and getting error: `TypeError: unknown type "Eip712" `

Reply:
let's try to put EIP712 here :)